pyvista/themes.py

```diff
@@ -2187,6 +2206,7 @@ def __init__(self):
         self.axes.x_color = 'tomato'
         self.axes.y_color = 'seagreen'
         self.axes.z_color = 'blue'
+        self.return_cpos = False
```

Comment:
Is there justification to also change the `document` theme?

Reply:
Yes.  I really don't like how whenever we generate docs we end up with a figure and our camera position when showing.  Only way around this is `cpos = plotter.show()`, which looks messy.

Reply:
Why can't we just add `pyvista.global_theme.return_cpos = False` in `doc/conf.py`? That would do exactly what's needed: affect our own doc building without seeping downstream. Changing the test theme is fine because that's only meant to be used internally.

Reply:
That makes more sense.  Done in 32866c751678f857dbbd77c105b6c5f072533b22.